docs.html

```diff
@@ -49,7 +49,19 @@
 <h2>
 <a name="migrating" class="anchor" href="#migrating"><span class="octicon octicon-link"></span></a>Migrating a module to puppet-community</h2>
 
-<p> TODO: add docs on how to migrate a module </p>
+<p>The general flow is to…</p>
+
+<ul>
+  <li>Ask one of the Administrators to add you as a collaborator.</li>
```

Comment:
This isn't sufficient. We need to create an admin group for the modules. We can create a collaborator/admins group if we want them all to be together.

Also we need a step for someone to add the colaborators group to the module so that other people can merge code, if that is desired.